packages/console/src/components/Steps/components/StepItem/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { type AdminConsoleKey } from '@logto/phrases';
+import { type ReactNode } from 'react';
+
+import DynamicT from '@/components/DynamicT';
+
+import Status, { type StepStatus } from './Status';
+import * as styles from './index.module.scss';
+
+export type StepItemProps = {
+  step: number;
+  currentStep: number;
+  status: StepStatus;
+  title: AdminConsoleKey;
+  content?: ReactNode;
+  isContentVisibleOnFinished?: boolean;
+  isDashLineVisible?: boolean;
```

Comment:
Is this a design requirement?

Reply:
`isContentVisibleOnFinished` is a design requirement
`isDashLineVisible` is use to hide the dash line for the last step item.